tests/post/features/monitoring.feature

```diff
@@ -56,3 +56,16 @@ Feature: Monitoring is up and running
     Scenario: Expected Grafana dashboards are available
         Given the Grafana API is available
         Then the deployed dashboards match the expected ones
+
+    Scenario: Inserting a new ConfigMap provides a new dashboard
+        Given the Grafana API is available
+        And the 'zenko' namespace exists
+        When we put a dashboard as ConfigMap named 'zk-dash' in namespace 'zenko' with folder annotation value 'ZENKO-PLEASE'
+        Then we have a dashboard named 'zk-dash' in folder 'ZENKO-PLEASE'
+
+    Scenario: Removing a ConfigMap removes its dasboard
+        Given the Grafana API is available
+        And the 'zenko' namespace exists
+        And the 'zk-dash' dashboard exists in folder 'ZENKO-PLEASE'
+        When we delete the 'zk-dash' ConfigMap in namespace 'zenko'
+        Then we have no dashboard named 'zk-dash' in folder 'ZENKO-PLEASE'
```

Comment:
Personally do not really like the fact that this test need the previous one to succeed.

To me we could have a single test with 2 "When - Then" (but maybe it's not a good practice I don't know)
Something like that 
```
Scenario: Inserting a new ConfigMap provides a new dashboard
    Given the Grafana API is available
    And the 'zenko' namespace exists
    When we put a dashboard as ConfigMap named 'zk-dash' in namespace 'zenko' with folder annotation value 'ZENKO-PLEASE'
    Then we have a dashboard named 'zk-dash' in folder 'ZENKO-PLEASE'
    When we delete the 'zk-dash' ConfigMap in namespace 'zenko'
    Then we have no dashboard named 'zk-dash' in folder 'ZENKO-PLEASE'
```

Reply:
```         
- And the 'zk-dash' dashboard exists in folder 'ZENKO-PLEASE'
+ And the 'zk-dash' ConfigMap exists in folder 'ZENKO-PLEASE'
`

Reply:
But with the suggestion, I send to you the CM will be deleted at the end of the test also.

If you just want some clean up at the end of the test it's doable with some teardown  (you can check `tests/post/steps/test_ingress.py` we do some simple teardown)